File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n = nums.length;
    let currentSum = 0;
    let expectedSum = n*(n+1)/2;
    for(let i = 0;i<n;i++){
        currentSum += nums[i];
    }
    return expectedSum - currentSum;
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int currentSum = 0;
        int n = nums.length;
        int expectedSum = n*(n + 1)/2;
        for(int i = 0;i<n;i++){
            currentSum+=nums[i];
        }
        return expectedSum - currentSum;

    }
}